addons/survey/static/src/js/fields_section_one2many.js

```diff
@@ -51,7 +51,11 @@ var SectionListRenderer = ListRenderer.extend({
                 if (record.data.questions_selection === "random") {
                     nbrColumns--;
                 }
+                // Render empty cells for buttons to avoid having unaligned elements
```

Comment:
I remember we discussed this change but I'm not sure what it's about anymore.
Can you explain again?

Reply:
It's for https://www.awesomescreenshot.com/image/11096552?key=127083eb310eefe2beaa8971caad44b0

When we activate the "Randomized per Section" option, there was a problem with the alignment because the section took too much space. By reducing the number of columns the section takes, we can create enough space pour the "Random Question Count" column the be properly alligned.